### ISSUE TITLE
getDownloadUrl fails with external URLs

### ISSUE THREAD

**Issue by ion1** [VeguiDev/spiget-api#1]
`getDownloadUrl` seems not to handle external URLs:

`$ node -e '(async () => {const SpigetAPI = require("spiget-api").SpigetAPI; const spiget = new SpigetAPI(); const res = await spiget.getResource(18494); console.log(res?.external, res?.file?.externalUrl); console.log(await res.getDownloadUrl()) })()'`
```
true https://lol.scarsz.me/RlLqpp/DiscordSRV-Build-1.26.0.jar
null
```

**Comment by VeguiDev** [comment]
Yes, this method only works if the resource is hosted on Github or on Spigot itself. For now the only way to access the url is using the file.externalUrl field.

Soon the getDownloadUrl method will support it.
